### PR TITLE
Adds callback: ReduceLRExponentialDecay

### PR DESCRIPTION
**Pull request by HiiYL**
I would like to contribute a new callback, created as part of my attempt in implementing the paper:
https://arxiv.org/pdf/1602.07261v2.pdf
it provides an exponential decay option for learning rate at the end of every n epochs.

### CODE REVIEW

**Comment by fchollet** [comment]
To avoid bloat, we will not be checking in this feature into core Keras. This is something that may be a better fit for a repo of Keras snippets / extensions.

**Comment by patyork** [comment]
Out of curiosity, is there any kind of designated repo for this sort of thing? Anything "official" or that you've seen kept up to data, and that is useful?

I can understand the need to separate additional activations, metrics, layers, example, etc. from the core, but I'd hope for a specific place to put them.

**Comment by fchollet** [comment]
There are multiple such repos, although I do not know whether they are well-maintained or used by any reasonable number of people. E.g.
https://github.com/wuaalb/keras_extensions
https://github.com/EderSantana/seya
https://github.com/anayebi/keras-extra

You could also consider starting your own, with a clean submission workflow for people to submit e.g. their custom callbacks, layers, etc. The most popular ones could then move to core Keras.

**Comment by carlthome** [comment]
How about having a `contrib` package in the official repo, like TensorFlow?
